### DISCUSSION TITLE
Is there generated pdf version notes?

### DISCUSSION THREAD

**Discussion by 0324wy** [mlc-ai/mlc-en#47] [Q&A]
Is there generated pdf version notes?

<img width="1440" alt="截屏2023-04-17 09 43 58" src="https://user-images.githubusercontent.com/48914068/232358713-48323b47-e13a-4d84-94cf-f818eda8f0c0.png">

**Comment by Hzfengsy** [comment]
Currently, we don't have a pdf version. Please check out our website. And any comments about the web pages are welcomed :)
